tf-module/stac_browser/variables.tf

```diff
@@ -0,0 +1,56 @@
+variable "prefix" {
+  type = string
+}
+variable "aws_region" {
+  type    = string
+  default = "us-west-2"
+}
+variable "tags" {
+  description = "Tags to be applied to Cumulus resources that support tags"
+  type        = map(string)
+  default     = {}
+}
+
+variable "cumulus_lambda_vpc_id" {
+  description = "VPC ID."
+  type = string
+}
+
+variable "subnet_ids" {
+  description = "subnet IDs for Lambdas"
+  type        = list(string)
+  default     = null
+}
+
+variable "ami_id" {
```

Comment:
@wphyojpl Can we read the ami_id from MCP ssm so that each time we deploy it refreshes with the latest patched ami?

Reply:
Sure. I can change that.. But do you want that constraint to be at this level or at "cumulus-template-deploy" level so that we can at least re-use this outside MCP?

Reply:
Good point. Let's keep it at cumulus-template-deploy.